packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -385,6 +396,7 @@ class RenderListWheelViewport
   }
 
   void _hasScrolled() {
```

Comment:
Since this class was a diff different in that it only needed to repaint when scrolling and needs to do more now, we should revise the users of this. For instance, setting offset needs to call this now but we don't need layout for things like setting diameterRatio and perspective. They should just call needs paint and semantics only.

Reply:
done.